packages/perspective/src/js/perspective.js

```diff
@@ -333,6 +331,8 @@ class ArrowColumnLoader extends TypeVisitor {
         return this;
     }
     // visitNull(type/*: Arrow.type.Null*/) {}
+    // TODO: Should this variable be an argument? 
+    // eslint-disable-next-line no-unused-vars
     visitBool(type/*: Arrow.type.Bool*/) {
```

Comment:
There are many of these below, are the `type` arguments really needed?

Reply:
No, likely not.  `@apache-arrow/es5` defines `TypeVisitor` which uses defines this variable in Typescript - but these are certainly vestigial wrt this module.